lib/salesforceapi-rest.rb

```diff
@@ -31,16 +31,16 @@ def initialize(refresh_token, metadata_uri, client_id, client_secret)
         @refresh_token = refresh_token
         @client_id = client_id
         @client_secret = client_secret
-        @metadata_uri = metadata_uri
-        @api_version = "v54.0"
+        @api_version = "57.0"
+        @metadata_uri = metadata_uri.gsub("{version}", @api_version)
         @ssl_port = 443  # TODO, right SF use port 443 for all HTTPS traffic.
 
       end
 
 
       def create(object, attributes)
         config_authorization!
-        path = "/services/data/#{@api_version}/sobjects/#{object}/"
+        path = "/services/data/v#{@api_version}/sobjects/#{object}/"
```

Comment:
Unsure if moving the "v" character here is best practice? Alternatively could create two variables above for "54.0" and "v54.0" but that seemed unnecessary.

Reply:
I think it is totally fine to put the `v` in the URL path. It's part of Salesforce's API so 👍